From 68d6aa65f43557de008abc7ec97856906bf5e657 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Sun, 21 Apr 2024 10:35:19 +0200
Subject: [PATCH] abuild: skip /var/run in fhs check

We have many package containing directories under /var/run. Allow those
for now.

Partially reverts commit 08d2d81587d9 (abuild: extend fhs check)
---
 abuild.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/abuild.in b/abuild.in
index f873620..0f70a4b 100644
--- a/abuild.in
+++ b/abuild.in
@@ -782,7 +782,7 @@ postcheck() {
 	msg "Running postcheck for $name"
 	# checking for FHS compat
 	if ! options_has "!fhs"; then
-		for i in srv usr/local opt var/run tmp var/tmp var/lock var/empty home sys proc mnt dev; do
+		for i in srv usr/local opt tmp var/tmp var/lock var/empty home sys proc mnt dev; do
 			for j in "$dir"/"$i"/* "$dir"/"$i"/.[!.]* "$dir"/"$i"/..?*; do
 				if [ -L "$j" ] || [ -e "$j" ]; then
 					error "Packages must not put anything under /$i"
-- 
2.44.0

